### ISSUE TITLE
Logo Design For LeafNotes

### ISSUE THREAD

**Issue by zularizal** [AreebMalik1989/LeafNotes#1]
Hi @AreebMalik1989 , I'm a graphic designer. I am interested in your project. That's why I want to design a free logo for your project. What do you think?

**Comment by AreebMalik1989** [comment]
Hi @zularizal, thanks a lot for your interest and you are most welcome for your contribution in this project!

**Comment by zularizal** [comment]
Hi, I'm sorry for making a long wait. I took a leave of absence for several months. today I started back. And this is the logo design for your project.
![logo](https://user-images.githubusercontent.com/38547278/64092390-cec37480-cd7e-11e9-932b-baef702534ef.png)

**Comment by AreebMalik1989** [comment]
Hi @zularizal , thanks a lot for your efforts, I'll update the project with these logo designs and add you to the contributors list ASAP.

I understand you must be busy, as I myself is hardly finding time for open source projects these days. Thanks again for your helping me with your designs.

**Comment by AreebMalik1989** [comment]
Hi @zularizal, I have updated readme.md file & application with logos you provided.
Also, I have added you in credit section of readme.md file.

Thanks for your contribution in this project, your design are extremely good and intuitive.
Soon I'll add credit section in app itself and will add you for your contribution.
Do let me know if I can do anything to give you more credit for your contribution in this project.
